kex-runner/src/main/kotlin/org/vorpal/research/kex/config/RunnerCmdConfig.kt

```diff
@@ -57,6 +57,16 @@ class RunnerCmdConfig(args: Array<String>) : AbstractCmdConfig("kex-runner", arg
     }")
     mode.isRequired = false
     options += mode
+
+    val minimization = Option(
```

Comment:
Command line options in Kex are used to pass parameters that are specific and necessary for current execution: e.g. class path, target name, launch mode, etc.  

For parameters that fall under the category of "configuration options" we use separate configuration file &mdash; [`kex.ini`](https://github.com/vorpal-research/kex/blob/master/kex.ini). Parameters are divided into categories, I think this parameter can be put in default 'kex' category.

Example of how these parameters can be accessed in code can be found [here](https://github.com/vorpal-research/kex/blob/464a27fc529caedc4e5f2010677d2b7090480dd5/kex-runner/src/main/kotlin/org/vorpal/research/kex/jacoco/CoverageReporter.kt#L519). We have a global `kexConfig` variable that is initialized at the start and can be accessed to receive current configuration. 

I think its better to move this parameter to `kex.ini`.